pkg/sfu/streamallocator/streamallocator.go

```diff
@@ -239,7 +242,11 @@ func (s *StreamAllocator) SetBandwidthEstimator(bwe cc.BandwidthEstimator) {
 }
 
 func (s *StreamAllocator) SetSendSideBWE(sendSideBWE *sendsidebwe.SendSideBWE) {
+	if sendSideBWE != nil {
+		sendSideBWE.OnCongestionStateChange(s.onCongestionStateChange)
+	}
 	s.sendSideBWE = sendSideBWE
```

Comment:
is it thread-safe here?

Reply:
Should be. It happens at construction time (just after construction). So, it should be fine. Will make sure it is okay.